sdk/src/azure/core/az_mqtt5_connection_hfsm.c

```diff
@@ -218,6 +240,13 @@ static az_result started(az_event_policy* me, az_event event)
           az_event_policy_send_inbound_event((az_event_policy*)this_policy, event));
       break;
 
+    case AZ_HFSM_EVENT_ERROR:
+    case AZ_HFSM_EVENT_TIMEOUT:
```

Comment:
What timeout would this be? Also, you'll need to check that it matches this HFSM's timeout (like below), because it could be for a different policy, which should just be ignored
> if (event.data == &this_policy->_internal.rpc_client_timer)